app/src/main/java/it/niedermann/nextcloud/deck/remote/helpers/SyncHelper.java

```diff
@@ -44,23 +48,35 @@ public SyncHelper(@NonNull ServerAdapter serverAdapter, @NonNull DataBaseAdapter
 
     // Sync Server -> App
     public <T extends IRemoteEntity> void doSyncFor(@NonNull final AbstractSyncDataProvider<T> provider) {
+        doSyncFor(provider, true);
+    }
+    public <T extends IRemoteEntity> void doSyncFor(@NonNull final AbstractSyncDataProvider<T> provider, boolean parallel) {
```

Comment:
The intention of the `parallel` argument is effectively that boards are synchronized one by one, but while synchronizing one board, the cards and other stuff get fetched parallel, correct?

I am wondering what the `isParallel` flag in `BoardDataProvider` is for, what's the difference or how are they supposed to play together?